bootcamp/tutorials/integration/semantic_search_with_milvus_and_voyageai.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>clear the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/#comment-1609395066'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>solved</p>